aci/resource_aci_fvaepg.go

```diff
@@ -173,11 +173,51 @@ func resourceAciApplicationEPG() *schema.Resource {
 				Optional: true,
 				Set:      schema.HashString,
 			},
-			"relation_fv_rs_node_att": &schema.Schema{
+			"relation_fv_rs_node_att": &schema.Schema{ 
```

Comment:
After our discussion yesterday, would it be an idea to consider making relation_fv_rs_bd as a required input?

Reply:
We had decided to leave it not required but document that it is.

Reply:
ok